benchmark/legged-robots-codegen.cpp

```diff
@@ -284,7 +289,7 @@ int main(int argc, char* argv[]) {
   // Forward pass timings
   for (unsigned int i = 0; i < T; ++i) {
     crocoddyl::Timer timer;
-    cg_ddp.forwardPass(0.5);
+    cg_ddp.forwardPass(0.005);
```

Comment:
Indeed, it seems you reduced alpha because a big `dt` value as commented above.

Reply:
`5e-3` is not big. If this time step doesn't work, it is because the problem is ill-defined, not because the time step is too high.

Reply:
I agree with it. I referred to `dt=5e-2` as "big" in my other comment (probably I read it wrong).